6-AdvancedScenarios/3-call-api-acrs/AppCreationScripts/sample.json

```diff
@@ -0,0 +1,89 @@
+{
+    "Sample": {
+        "Title": "Use the Conditional Access auth context to perform step-up authentication for high-privilege operations in a web API",
+        "Level": 400,
+        "Client": "React SPA",
+        "Service": "Express web API",
+        "RepositoryUrl": "ms-identity-javascript-react-tutorial",
+        "Endpoint": "AAD v2.0"
+    },
+    "AADApps": [
+        {
+            "Id": "service",
+            "Name": "msal-node-api-acrs",
+            "Kind": "WebApi",
+            "Audience": "AzureADMyOrg",
+            "HomePage": "http://localhost:5000/admin/home",
+            "ReplyUrls": "http://localhost:5000/admin/redirect",
+            "PasswordCredentials": "Auto",
```

Comment:
>PasswordCredentials [](http://example.com/codeflow?start=13&length=19)

Delegated permissions of Policy.Read.ConditionalAccess, Policy.ReadWrite.ConditionalAccess

Reply:
Added!